SpreadCheetah/MetadataXml/WorksheetStartXml.cs

```diff
@@ -88,8 +88,15 @@ public bool MoveNext()
 
     private readonly bool TryWriteSheetViewsStart()
     {
-        return options is null or { FrozenColumns: null, FrozenRows: null }
-            || buffer.TryWrite("<sheetViews><sheetView workbookViewId=\"0\"><pane "u8);
+        if (options is null or { FrozenColumns: null, FrozenRows: null, ShowGridLines: null })
+            return true;
+
+        return options.ShowGridLines switch
+        {
+            true => buffer.TryWrite("<sheetViews><sheetView showGridLines=\"1\" workbookViewId=\"0\"><pane "u8),
+            false => buffer.TryWrite("<sheetViews><sheetView showGridLines=\"0\" workbookViewId=\"0\"><pane "u8),
+            null => buffer.TryWrite("<sheetViews><sheetView workbookViewId=\"0\"><pane "u8)
```

Comment:
Here we start the `<pane>` element in all cases, but I don't think that is correct in the case when both `FrozenColumns` and `FrozenRows` are `null`.